### ISSUE TITLE
Add support for Magento 2

### ISSUE THREAD

**Issue by punkstar** [meanbee/magedbm#34]
I'll just leave this here 😄

**Comment by brideo** [comment]
McBrides on  it.

**Comment by bobbyshaw** [comment]
Get me a shout if you want to chat about the approach.

**Comment by punkstar** [comment]
@brideo What's the status on this?  I see you've made a lot of progress on your fork.

**Comment by brideo** [comment]
Yeah I haven't forgot about this, feel free to pick it up I'm just a bit flat out. The PutCommand still needs some work. I'm not entirely happy with the stripping implementation as it's just an array of table names, n98 pretty much  do the same just with a yaml file.

https://github.com/brideo/magedbm/tree/magento2-support/src/Meanbee/Magedbm

The factory logic could also be improved but MVP bro.

**Comment by brideo** [comment]
I can schedule to smash some stuff out next week, integration tests would be pretty cool.

**Comment by bobbyshaw** [comment]
@brideo Yo, where we at? Got anything worth salvaging if we're looking into this again?

**Comment by brideo** [comment]
@bobbyshaw hey mate, sorry I haven't made any progress since my last comment. 

https://github.com/brideo/magedbm/commits/magento2-support

I think Magerun is still a dependency. 

I got pretty far so I would use my fork, do you want me to  submit a PR or will you just scrap it and start again?

**Comment by bobbyshaw** [comment]
I'd definitely look over it and try and use it ;) Submit a PR and I'll get it accepted into a new branch for us to play with.
